actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -31,6 +31,9 @@ class Railtie < Rails::Railtie # :nodoc:
     config.action_dispatch.debug_exception_log_level = :fatal
     config.action_dispatch.strict_freshness = false
 
+    config.action_dispatch.ignore_leading_brackets = nil
+    config.action_dispatch.strict_query_string_separator = nil
```

Comment:
Should these configs be documented? My understanding is that you should use them if you're upgrading from Rack 2 to Rack 3 but want to preserve Rack 2 behaviour for a little while longer. And also that they will both be removed in Rails 8.1?